.gitignore

```diff
@@ -1,3 +1,157 @@
-node_modules/
-dist/
+### WebStorm ###
```

Comment:
where did you pull this .gitignore from? My intuition is to use the [github Node one](https://github.com/github/gitignore/blob/master/Node.gitignore), which covers more Node-specific things